app/services/external_upload_manager.rb

```diff
@@ -196,6 +196,7 @@ def download(key, type)
       max_file_size: DOWNLOAD_LIMIT,
       tmp_file_name: "discourse-upload-#{type}",
       follow_redirect: true,
+      validate_uri: false,
```

Comment:
TODO: Had to do this because FinalDestination enforces certain ports based on URI scheme, probably a better way.